lib/rubocop/target_finder.rb

```diff
@@ -3,6 +3,53 @@
 require 'set'
 
 module RuboCop
+  RUBY_EXTENSIONS = %w(.rb
```

Comment:
I wonder if all these should be hardcoded instead of moved to the configuration. That would make it possible for users to disable RuboCop for certain types of files easily.